examples/seismic/skew_self_adjoint/ssa_01_iso_implementation.ipynb

```diff
@@ -0,0 +1,1452 @@
+      "text/plain": [
+       "<Figure size 864x576 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# note: flip sense of second dimension to make the plot positive downwards\n",
+    "plt_extent = [origin_pad[0], origin_pad[0] + spacing[0]*(shape_pad[0]-1), \n",
+    "              origin_pad[1] + spacing[1]*(shape_pad[1]-1), origin_pad[1]]\n",
+    "\n",
+    "plt.figure(1, figsize=(12,8))\n",
+    "plt.clf()\n",
+    "plt.imshow(np.transpose(v.data), cmap=cm.jet, vmin=1.25, vmax=1.75, extent=plt_extent)\n",
+    "plt.plot([origin[0], origin[0], extent[0], extent[0], origin[0]], \n",
+    "         [origin[1], extent[1], extent[1], origin[1], origin[1]], \n",
+    "         'white', linewidth=4, linestyle=':', label=\"Absorbing Boundary\")\n",
+    "plt.plot(rec.coordinates.data[:, 0], rec.coordinates.data[:, 1], \\\n",
+    "         'blue', linewidth=5, linestyle='--', label=\"Receiver line\")\n",
+    "plt.plot(src.coordinates.data[:, 0], src.coordinates.data[:, 1], \\\n",
+    "         'red', linestyle='None', marker='*', markersize=15, label=\"Source\")\n",
+    "ax = plt.legend(bbox_to_anchor=(1.25, 0.5), loc='upper left')\n",
+    "ax.get_frame().set_color(\"gray\")\n",
+    "plt.colorbar()\n",
+    "plt.xlabel(\"X Coordinate (m)\", labelpad=15)\n",
+    "plt.ylabel(\"Z Coordinate (m)\", labelpad=15)\n",
+    "plt.title(\"Wholespace velocity model including absorbing boundary region\", y=1.025)\n",
```

Comment:
I love this plot!

Reply:
thanks, I always like to think about the ABC. it looks different now since I changed to a wholespace.